gluster-exporter/metric_volume.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	volumeHealLabels = []MetricLabel{
+		{
+			Name: "volume",
+			Help: "Volume Name",
+		},
+		{
+			Name: "brick_path",
+			Help: "Brick Path",
+		},
+		{
+			Name: "host",
+			Help: "Hostname",
+		},
+	}
+
+	glusterVolumeHealCount = newPrometheusGaugeVec(Metric{
+		Namespace: "gluster",
+		Name:      "volume_heal_count",
+		Help:      "self heal count for volume",
+		LongHelp:  "",
+		Labels:    volumeHealLabels,
+	})
+)
+
+func getVolumeHealLabels(volname string, host string, brick string) prometheus.Labels {
+	return prometheus.Labels{
+		"volume":     volname,
+		"brick_path": brick,
+		"host":       host,
+	}
+
+}
+
+func healCounts() error {
+	isLeader, err := gluster.IsLeader()
+	if !isLeader || err != nil {
+		// Unable to find out if the current node is leader
+		// Cannot register volume metrics at this node
+		log.WithError(err).Error("Unable to find if the current node is leader")
+		return err
+	}
+	volumes, err := gluster.VolumeInfo()
+	if err != nil {
+		return err
+	}
+
+	for _, volume := range volumes {
+		name := volume.Name
+		heals, err := gluster.HealInfo(name)
+		if err != nil {
+			log.WithError(err).WithFields(log.Fields{
+				"volume": name,
+			}).Error("Error getting disk usage")
```

Comment:
disk usage or heal info?

Reply:
it is typo, Yes it is heal info. I will fix this.